configure

```diff
@@ -4112,6 +4116,36 @@ fi
 
 fi
 
+# REPORTS_BASE_URL definition
+
+
+
+# Check whether --with-reports-base-url was given.
+if test "${with_reports_base_url+set}" = set; then :
```

Comment:
`base_url` is sort of weird. Can we do `with_reports_hostname`?

Reply:
Done.